src/org/ods/services/BitbucketService.groovy

```diff
@@ -1,9 +1,15 @@
 package org.ods.services
 
 import org.ods.util.ILogger
+import com.cloudbees.groovy.cps.NonCPS
+import org.ods.util.AuthUtil
 
+@SuppressWarnings('PublicMethodsBeforeNonPublicMethods')
```

Comment:
Is this even needed now? Looks like the order adheres to "public before non-public" now ...

Reply:
Without it it complains about: 

Map<String, String> createUserToken() {
[MSG]The public method createUserToken in class org.ods.services.BitbucketService is declared after a non-public method

And then is over the count,  I guess I would have to diff reports in order to figure out where another prio 3 issue came in.

Reply:
OK, then we leave it like it is ...